examples/06_Combining_Models.ipynb

```diff
@@ -6,6 +6,250 @@
    "source": [
     "# Combining Prognostic Models"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section demonstrates how prognostic models can be combined. There are two times in which this is useful: \n",
+    "\n",
+    "1. When combining multiple models of different inter-related systems into one system-of-system model (i.e., [Composite Models](https://nasa.github.io/progpy/api_ref/prog_models/CompositeModel.html)), or\n",
+    "2. Combining multiple models of the same system to be simulated together an aggregated (i.e., [Ensemble Models](https://nasa.github.io/progpy/api_ref/prog_models/EnsembleModel.html)). This is generally done to improve the accuracy of prediction when you have multiple models that each represent part of the behavior or represent a distribution of different behaviors. \n",
+    "\n",
+    "These two methods for combining models are described in the following sections."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Composite Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A CompositeModel is a PrognosticsModel that is composed of multiple PrognosticsModels. This is a tool for modeling system-of-systems. i.e., interconnected systems, where the behavior and state of one system effects the state of another system. The composite prognostics models are connected using defined connections between the output or state of one model, and the input of another model. The resulting CompositeModel behaves as a single model.\n",
+    "\n",
+    "To illustrate this, we will create a compositemodel of an aircraft's electric powertrain, combining the DCMotor, ESC, and PropellerLoad models.\n",
+    "\n",
+    "First we will import the used models, and the CompositeModel class"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy.models import DCMotor, ESC, PropellerLoad\n",
+    "from progpy import CompositeModel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we will initiate objects of the individual models that will later create the composite powertrain model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m_motor = DCMotor()\n",
+    "m_esc = ESC()\n",
+    "m_load = PropellerLoad()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we have to define the connections between the systems. Let's first define the connections from the DCMotor to the propeller load."
```

Comment:
Suggestion to add to the end, just so it's clear, "For this, we'll need to look at the DCMotor states and understand how they influence the PropellerLoad inputs".